content/general/concepts/Yukihiro_Matsumoto/Yukihiro_Matsumoto.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Yukihiro Matsumoto (Matz)'
+Description: 'Yukihiro Matsumoto a.k.a one of the open source evangelists of Japan, is a Japanese Software Programmer and Computer Scientist widely known for his work as the chief designer of Ruby. Matz blended parts of his favorite languages (Perl, Smalltalk, Eiffel, Ada, and Lisp) to form Ruby, a language of careful balance. 
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Ruby'
+  - 'Perl'
+CatalogContent:
+  - 'paths/code-foundations'
+---
+
+![Yukihiro Matsumoto (Matz)](https://raw.githubusercontent.com/Codecademy/docs/main/media/Yukihiro_Matsumoto.png
+
+**Yukihiro Matsumoto** (1965-present) a.k.a one of the open source evangelists of Japan, is a Japanese Software Programmer and Computer Scientist widely known for his work as the chief designer of Ruby. Matz blended parts of his favorite languages (Perl, Smalltalk, Eiffel, Ada, and Lisp) to form [Ruby](https://www.codecademy.com/resources/docs/ruby), a language of careful balance. Apart from Ruby, he is known for having released several open source products, including cmail, the emacs-based mail user agent, written entirely in emacs lisp.
+
+As of 2011, Matz is the Chief Architect of Ruby at [Heroku](https://en.wikipedia.org/wiki/Heroku "Heroku"), an online cloud platform-as-a-service in San Francisco. He was appointed to the role of technical advisor for VASILY Inc. starting in June 2014.
+
+## Writing
+
+In 2001, Matz published a book called _Ruby in a Nutshell_, and in 2008 he published _The Ruby Programming Language_. His demeanor has brought about a motto in the Ruby community: _"Matz is nice and so we are nice,"_ commonly abbreviated as MINASWAN.
```

Comment:
```suggestion
In 2001, Matz published a book called "Ruby in a Nutshell." This was later expanded and revised into a new book called "The Ruby Programming Language," released in 2008 and co-authored by David Flanagan.
```

With the exception of the first mention of the entry's concept/term, let's refrain from using emphasis and, instead, wrap a published work in quotes.

Also, I think the last sentence fits better under the opening sentences.